### PR TITLE
8254850: Update terminology in java.awt.GridBagLayout source code comments

### PR DESCRIPTION
**Pull request by prrace**
This is part of an effort in the JDK to replace archaic/non-inclusive words with more neutral terms.
See JDK-8253315 for details.
 
This updates a few comments in GridBagLayout to use the terms parent and child which are
sufficiently descriptive and consistent with lots of other usages.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8254850](https://bugs.openjdk.java.net/browse/JDK-8254850): Update terminology in java.awt.GridBagLayout source code comments

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - Author) ⚠️ Review applies to 60eeff0555302515f4f393ffb43e3aa9005e9b79
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to 60eeff0555302515f4f393ffb43e3aa9005e9b79

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/1784/head:pull/1784`
`$ git checkout pull/1784`

### CODE REVIEW

File: src/java.desktop/share/classes/java/awt/GridBagLayout.java [hunk 1]
```diff
@@ -1364,7 +1364,7 @@ else if (constraints.gridwidth == 0 && curCol < 0)
                         px = constraints.tempX + constraints.tempWidth; /* right column */
 
                         /*
-                         * Figure out if we should use this slave\'s weight.  If the weight
+                         * Figure out if we should use this child\'s weight.  If the weight
```
Comment (kevinrushforth) [inline src/java.desktop/share/classes/java/awt/GridBagLayout.java:1367]: Is the `\` still needed?

**Review by kevinrushforth — APPROVED** [review]
Looks good. One minor question.

**Review by azuev-java — APPROVED** [review]
Looks good.

File: src/java.desktop/share/classes/java/awt/GridBagLayout.java (hunk 1 above)
Comment (prrace) [reply src/java.desktop/share/classes/java/awt/GridBagLayout.java:1367]: Not for any reason I can imagine.

**Pull request closed without merging**